### PR TITLE
Implement uploading avatar for user and BP

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**

### CODE REVIEW

File: directory/block_producer/models.py [hunk 1]
```diff
@@ -104,6 +108,8 @@ def get(cls, identifier):
         """
         block_producer_as_dict = cls.objects.filter(id=identifier).values().first()
 
+        print(block_producer_as_dict)
```
Comment (anastasiia-bilova) [inline directory/block_producer/models.py]: Remove debug code.

File: directory/block_producer/views/avatar.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+from django.http import JsonResponse
+from rest_framework.decorators import (
+    authentication_classes,
+    permission_classes,
+)
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+
+
```
Comment (anastasiia-bilova) [inline directory/block_producer/views/avatar.py]: Remove blank line.

File: directory/block_producer/views/avatar.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        self.block_producer = BlockProducer()
+
+    @authentication_classes((JSONWebTokenAuthentication, ))
+    def post(self, request, block_producer_id):
+        """
+        Upload block producer avatar.
+        """
+        user_email = request.user.email
+
+        form = UploadUserAvatarForm(request.POST, request.FILES)
```
Comment (anastasiia-bilova) [inline directory/block_producer/views/avatar.py]: Use block producers form.

File: directory/services/avatar.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
+"""
+Provide implementation of avatar.
+"""
+from enum import (
+    Enum,
+    auto,
+)
+
+import boto3
+
```
Comment (anastasiia-bilova) [inline directory/services/avatar.py]: Remove blank line.

**Review by anastasiia-bilova — APPROVED** [review]

File: directory/block_producer/models.py (hunk 1 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/models.py]: @anastasiia-bilova, done.

File: directory/block_producer/views/avatar.py (hunk 2 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/views/avatar.py]: @anastasiia-bilova, done.

File: directory/block_producer/views/avatar.py (hunk 3 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/views/avatar.py]: @anastasiia-bilova, done.

File: directory/services/avatar.py (hunk 4 above)
Comment (dmytrostriletskyi) [reply directory/services/avatar.py]: @anastasiia-bilova, done.

**Pull request merged**